<?xml version="1.0" encoding="UTF-8" ?>
<testsuite tests="10" failures="8" name="testquestion2.Test_MyCollection" time="0.376" errors="0" skipped="0">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="/usr/lib/jvm/java-8-oracle/jre/lib/amd64"/>
    <property name="java.vm.version" value="25.171-b11"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=":"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.country" value="IN"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value="unknown"/>
    <property name="test" value="testquestion2.Test_MyCollection"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/home/arpitagarwal/task_9July18/qa-test"/>
    <property name="java.runtime.version" value="1.8.0_171-b11"/>
    <property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/>
    <property name="java.endorsed.dirs" value="/usr/lib/jvm/java-8-oracle/jre/lib/endorsed"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="/tmp"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="os.name" value="Linux"/>
    <property name="maven.ext.class.path" value="/home/arpitagarwal/netbeans-8.2/java/maven-nblib/netbeans-eventspy.jar"/>
    <property name="classworlds.conf" value="/home/arpitagarwal/netbeans-8.2/java/maven/bin/m2.conf"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/usr/lib/jvm/java-8-oracle/jre/lib/amd64:/usr/lib/jvm/java-8-oracle/jre/lib/i386::/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="52.0"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="4.4.0-130-generic"/>
    <property name="user.home" value="/home/arpitagarwal"/>
    <property name="user.timezone" value="Asia/Kolkata"/>
    <property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/>
    <property name="java.specification.version" value="1.8"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="user.name" value="QAIT\arpitagarwal"/>
    <property name="java.class.path" value="/home/arpitagarwal/netbeans-8.2/java/maven/boot/plexus-classworlds-2.4.jar"/>
    <property name="java.vm.specification.version" value="1.8"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="java.home" value="/usr/lib/jvm/java-8-oracle/jre"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher -Dtest=testquestion2.Test_MyCollection -Dmaven.ext.class.path=/home/arpitagarwal/netbeans-8.2/java/maven-nblib/netbeans-eventspy.jar surefire:test"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="en"/>
    <property name="awt.toolkit" value="sun.awt.X11.XToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.8.0_171"/>
    <property name="java.ext.dirs" value="/usr/lib/jvm/java-8-oracle/jre/lib/ext:/usr/java/packages/lib/ext"/>
    <property name="securerandom.source" value="file:/dev/./urandom"/>
    <property name="sun.boot.class.path" value="/usr/lib/jvm/java-8-oracle/jre/lib/resources.jar:/usr/lib/jvm/java-8-oracle/jre/lib/rt.jar:/usr/lib/jvm/java-8-oracle/jre/lib/sunrsasign.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jsse.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jce.jar:/usr/lib/jvm/java-8-oracle/jre/lib/charsets.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jfr.jar:/usr/lib/jvm/java-8-oracle/jre/classes"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="maven.home" value="/home/arpitagarwal/netbeans-8.2/java/maven"/>
    <property name="file.separator" value="/"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.desktop" value="gnome"/>
    <property name="sun.cpu.isalist" value=""/>
  </properties>
  <testcase classname="testquestion2.Test_MyCollection" name="Test10_Rotate_Array" time="0.014">
    <failure message="expected [{Is, Always, A Fun., Sometimes, Automation, Testing}] but found [question2.MyCollection@9f70c54]" type="java.lang.AssertionError">java.lang.AssertionError: expected [{Is, Always, A Fun., Sometimes, Automation, Testing}] but found [question2.MyCollection@9f70c54]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at testquestion2.Test_MyCollection.Test10_Rotate_Array(Test_MyCollection.java:94)
</failure>
  </testcase>
  <testcase classname="testquestion2.Test_MyCollection" name="Test1_Able_To_Add_New_Entry_In_Collection" time="0.001">
    <failure message="Assertion Failed : Not able to add number 5 in collection expected [true] but found [false]" type="java.lang.AssertionError">java.lang.AssertionError: Assertion Failed : Not able to add number 5 in collection expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertTrue(Assert.java:44)
	at testquestion2.Test_MyCollection.Test1_Able_To_Add_New_Entry_In_Collection(Test_MyCollection.java:16)
</failure>
  </testcase>
  <testcase classname="testquestion2.Test_MyCollection" name="Test2_Duplicate_Entry_In_Collection" time="0.002"/>
  <testcase classname="testquestion2.Test_MyCollection" name="Test3_Get_Collection_Functionality" time="0.002">
    <failure message="Assertion Failed : Not able to read collection expected [{}] but found [question2.MyCollection@61a485d2]" type="java.lang.AssertionError">java.lang.AssertionError: Assertion Failed : Not able to read collection expected [{}] but found [question2.MyCollection@61a485d2]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at testquestion2.Test_MyCollection.Test3_Get_Collection_Functionality(Test_MyCollection.java:32)
</failure>
  </testcase>
  <testcase classname="testquestion2.Test_MyCollection" name="Test4_DoubleCapacity" time="0.002">
    <failure message="Assertion Failed : Not able to add number in collection once it get fulled expected [{4,5,-4,-5,6}] but found [question2.MyCollection@7946e1f4]" type="java.lang.AssertionError">java.lang.AssertionError: Assertion Failed : Not able to add number in collection once it get fulled expected [{4,5,-4,-5,6}] but found [question2.MyCollection@7946e1f4]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at testquestion2.Test_MyCollection.Test4_DoubleCapacity(Test_MyCollection.java:41)
</failure>
  </testcase>
  <testcase classname="testquestion2.Test_MyCollection" name="Test5_Remove_Functionality_If_Entry_Already_Present" time="0.003">
    <failure message="Assetion Failed : Not Able to delete entry from collection expected [true] but found [false]" type="java.lang.AssertionError">java.lang.AssertionError: Assetion Failed : Not Able to delete entry from collection expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertTrue(Assert.java:44)
	at testquestion2.Test_MyCollection.Test5_Remove_Functionality_If_Entry_Already_Present(Test_MyCollection.java:50)
</failure>
  </testcase>
  <testcase classname="testquestion2.Test_MyCollection" name="Test6_Remove_Functionality_If_Collection_Is_Empty" time="0.001"/>
  <testcase classname="testquestion2.Test_MyCollection" name="Test7_Collection_Length_Functionality" time="0.002">
    <failure message="expected [6] but found [0]" type="java.lang.AssertionError">java.lang.AssertionError: expected [6] but found [0]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:652)
	at org.testng.Assert.assertEquals(Assert.java:662)
	at testquestion2.Test_MyCollection.Test7_Collection_Length_Functionality(Test_MyCollection.java:66)
</failure>
  </testcase>
  <testcase classname="testquestion2.Test_MyCollection" name="Test8_Search_Functionality_If_Enrty_Exists" time="0.003">
    <failure message="expected [1] but found [0]" type="java.lang.AssertionError">java.lang.AssertionError: expected [1] but found [0]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:652)
	at org.testng.Assert.assertEquals(Assert.java:662)
	at testquestion2.Test_MyCollection.Test8_Search_Functionality_If_Enrty_Exists(Test_MyCollection.java:75)
</failure>
  </testcase>
  <testcase classname="testquestion2.Test_MyCollection" name="Test9_Search_Functionality_If_Enrty_Not_Exists" time="0.001">
    <failure message="expected [-1] but found [0]" type="java.lang.AssertionError">java.lang.AssertionError: expected [-1] but found [0]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:652)
	at org.testng.Assert.assertEquals(Assert.java:662)
	at testquestion2.Test_MyCollection.Test9_Search_Functionality_If_Enrty_Not_Exists(Test_MyCollection.java:84)
</failure>
  </testcase>
</testsuite>